Demo/ColorSliderDemo/DemoView.swift

```diff
@@ -108,7 +108,7 @@ class DemoView: UIView {
 		NSLayoutConstraint.activate([
 			colorSlider.centerXAnchor.constraint(equalTo: centerXAnchor),
 			colorSlider.bottomAnchor.constraint(equalTo: centerYAnchor),
-			colorSlider.widthAnchor.constraint(equalToConstant: 15),
+			colorSlider.widthAnchor.constraint(equalToConstant: 40),
```

Comment:
Please revert this change, the `DemoView`'s `ColorSlider` is intentionally 15 points wide.

Reply:
fixed